main.go

```diff
@@ -15,29 +17,40 @@ import (
 	"github.com/square/beancounter/deriver"
 	. "github.com/square/beancounter/utils"
 	"gopkg.in/alecthomas/kingpin.v2"
-	//	_ "net/http/pprof"
 )
 
-// TODO: create sub commands:
-// - findAddr
-// - findBlock
-// - watchMultiSig
-// - watch
-// - singleAddress
 var (
-	m             = kingpin.Flag("m", "number of signatures (quorum)").Short('m').Required().Int()
-	n             = kingpin.Flag("n", "number of public keys").Short('n').Required().Int()
-	account       = kingpin.Flag("account", "account number").Required().Uint32()
-	backendName   = kingpin.Flag("backend", "electrum | btcd | electrum-recorder | btcd-recorder | fixture").Default("electrum").Enum("electrum", "btcd", "electrum-recorder", "btcd-recorder", "fixture")
-	lookahead     = kingpin.Flag("lookahead", "lookahead size").Default("100").Uint32()
-	addr          = kingpin.Flag("addr", "backend to connect to initially").PlaceHolder("HOST:PORT").TCP()
-	rpcUser       = kingpin.Flag("rpcuser", "RPC username").PlaceHolder("USER").String()
-	rpcPass       = kingpin.Flag("rpcpass", "RPC password").PlaceHolder("PASSWORD").String()
-	debug         = kingpin.Flag("debug", "debug output").Default("false").Bool()
-	findAddr      = kingpin.Flag("find-addr", "finds the offset of an address").String()
-	blockHeight   = kingpin.Flag("block-height", "compute balance at given block height. Defaults to current chain height - 6.").Default("0").Uint32()
-	singleAddress = kingpin.Flag("single-address", "for debugging purpose").String()
-	fixtureFile   = kingpin.Flag("fixture-file", "fixture file to use for recording data or reading from").PlaceHolder("FILEPATH").String()
+	app   = kingpin.New("beancounter", "A command-line Bitcoin wallet balance audit tool.")
+	debug = app.Flag("debug", "Enable debug output.").Default("false").Bool()
+
+	keytree    = app.Command("keytree", "Performs one or more child key derivations.")
+	keytreeArg = keytree.Arg("i", "(repeated) Values for path.").Required().Uint32List()
+	keytreeN   = keytree.Flag("n", "number of public keys").Short('n').Default("1").Int()
+
+	findAddr    = app.Command("find-addr", "Finds the change/index values for a given address.")
```

Comment:
nit: should we just call it `find-address` for better readability of commands?

Reply:
sure